sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -510,12 +511,13 @@ def read_porttab_mappings(self, porttabfile, asic_inst=0):
                 elif "asic_port_name" not in title and len(line.split()) >= 4:
                     fp_port_index = int(line.split()[3])
                 else:
-                    fp_port_index = portname.split("Ethernet").pop()
+                    front_panel_prefix = re.search(FRONT_PANEL_PORT_PREFIX_REGEX, portname).group(1)
```

Comment:
You should check group size before accessing (1)? (same for line 519)

Reply:
yes, I'll fix it, thanks!